mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/impl/BatchIndexingWorkspace.java

```diff
@@ -29,19 +30,19 @@
  *
  * @author Sanne Grinovero
  */
-public class BatchIndexingWorkspace extends ErrorHandledRunnable {
+public class BatchIndexingWorkspace<E, I> extends ErrorHandledRunnable {
```

Comment:
Very nice having these generics here. Even if their meaning should be very compressible, maybe we could depict them in the java doc. The same you did for `IdentifierConsumerDocumentProducer`.